source/renderer/app/components/wallet/settings/UndelegateWalletConfirmationResultDialog.js

```diff
@@ -40,7 +40,7 @@ type Props = {
 type State = { timeUntilNextEpochStart: number };
 
 @observer
-export default class UndelegateConfirmationResultDialog extends Component<
+export default class UndelegateWalletConfirmationResultDialog extends Component<
```

Comment:
@yakovkaravelov can we remove "Result" from class name

Reply:
@tomothespian There is already other component which is named `UndelegateWalletConfirmationDialog`.
This one is for result.